index.html

```diff
@@ -3,32 +3,38 @@
 
   SPDX-License-Identifier: MIT
  -->
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en" style="height: 100%">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width,initial-scale=1" />
 
-    <title>micro:bit machine learning tool</title>
-    <meta
-      property="og:title"
-      content="machine learning tool - BBC micro:bit - the next gen" />
+    <title>micro:bit AI creator</title>
+    <meta property="og:title" content="micro:bit AI creator" />
     <link rel="canonical" href="%VITE_FULL_URL%" />
     <meta property="og:url" content="%VITE_FULL_URL%" />
     <meta name="twitter:card" content="summary_large_image" />
     <meta name="twitter:site" content="@microbit_edu" />
     <meta
       name="description"
-      content="Use our free machine learning tool as part of BBC micro:bit - the next gen's playground survey investigation. Explore machine learning, by training a computer model with your own data, then testing it to see if it works." />
+      content="Use our free micro:bit AI creator. Explore machine learning, by training a computer model with your own data, then testing it to see if it works."
```

Comment:
At some point we should redo this and other social metadata and consider capturing the MakeCode aspect.